app/models/course.rb

```diff
@@ -372,7 +372,7 @@ def scholarship_amount
   end
 
   def to_s
-    "#{name} (#{course_code})"
+    "[#{course_code}] #{name} (#{recruitment_cycle})"
```

Comment:
Can we keep the course code at the end? `"#{name} (#{course_code})"` is the consistent way we display courses everywhere.

Maybe:
`"#{name} (#{course_code}) [#{recruitment_cycle}]"`?

Reply:
(Or front load the recruitment cycle)

Reply:
I find that hard to read, software and code generally tends to place the unique code before details roughly in order of importance (unless it's extremely long, like a hash or memory address), and I've already merged in [a change](https://github.com/DFE-Digital/manage-courses-backend/pull/748) similar to this.

Reply:
What about copying the slash as it's in the URL?

```
"#{name} (#{recruitment_cycle}/#{course_code})"
```

Reply:
The slash is already being used for the provider code, putting the recruitment cycle there would look like (2019/20/QM1)

Reply:
Is that bad? That's still easy to parse:

```
course_parts = codes.s

Reply:
I only bring this up because I've wanted to try to normalise before, it'd be handy for supplying to `mcb`, for example ... you could provider multiple courses on the cmdline if we had a standard way to serialise their identities like this.

Reply:
This is for debug, so my concern here is readability. I don't think anything should be trying to parse this.